docs/user-guide/advanced-usage.rst

```diff
@@ -106,9 +122,14 @@ Then just call ``request`` as described above::
     auth,
     host="10.10.0.1",
     port=9090,
-    base_path="api/some_service",
+    base_path="api/some_service/",
   )
-  response = custom_client.get('some/specific/endpoint')
+  response = custom_client.get('relative/path/to/resource')
```

Comment:
In addition to this, can we provide a specific example? Like hitting GET /api/service/health ? Both using the relative and the absolute path?